src/Kernel/Object.class.st

```diff
@@ -359,6 +359,16 @@ Object >> asLinkPrepend: anObject [
 	^ self asLink asLinkPrepend: anObject
 ]
 
+{ #category : #converting }
```

Comment:
May we can already **package it into "Deprecated 11" package as an extension** as we later remove this package (and so we do not forget to remove this method)

So put it into *Deprecated11 category

Reply:
It is already into Deprecated11 package. See the last commit

Reply:
Cool - thanks!